map/gps_tracker.hpp

```diff
@@ -28,6 +30,11 @@ class GpsTracker
 
   void OnLocationUpdated(location::GpsInfo const & info);
 
+  bool IsRecentTrackCollectionInitialized() const;
```

Comment:
I don't think that this method is needed, considering that there is already IsEmpty method.

@kirylkaveryn let's remove it and merge the PR.